### DISCUSSION TITLE
Date / Time

### DISCUSSION THREAD

**Discussion by hugopvl** [directus/directus#6790] [Archive]
Good afternoon,

I have a problem with the registration dates and times and I don't understand what's going on:
when I select a date, July 14th, 2021, when I automatically record the date it changes to July 13th, 2021.

when I edit an item directly in directus the uptade date is correct, but in the database it is less than one hour.

![data_update](https://user-images.githubusercontent.com/81326296/125641106-3db9ba85-5d9a-4464-b6ef-ad3c520e55f2.PNG)

![data_update2](https://user-images.githubusercontent.com/81326296/125641120-c044f5a0-314b-4ae8-bdb1-cf8c53e144da.PNG)

**Comment by rijkvanzanten** [comment]
Are you using a datetime or a timestamp field? The hour difference is most likely a timezone difference

**Reply by hugopvl** [reply]
Hi @rijkvanzanten  im using datetime:
![image](https://user-images.githubusercontent.com/81326296/125641799-b4dfcc52-81bc-40c6-833d-15730fdad965.png)

where can i change the timezone?

**Reply by rijkvanzanten** [reply]
The DateTime field should save whatever value is submitted as is. Where is your database hosted?

**Reply by hugopvl** [reply]
Yes that's why I don't understand, the database is on a local server in sqlserver. I've already checked the server date and even on sqlserver and it's correct

**Reply by rijkvanzanten** [reply]
What's the server time of your local sql server / computer? 🤔

**Reply by hugopvl** [reply]
i think its all here 

server time 

![image](https://user-images.githubusercontent.com/81326296/125653850-fffdc8d0-37e5-4897-82d9-c6f235c8c205.png)

SQL 
![image](https://user-images.githubusercontent.com/81326296/125653983-3c53bf21-ee9b-4d3e-a3b7-cdf146e3e10d.png)

directus 

![image](https://user-images.githubusercontent.com/81326296/125654355-12ccd8e7-56ab-48ba-9ad8-8b8feaa176bb.png)

**Reply by rijkvanzanten** [reply]
Hmm that's odd. DateTime fields should send up just the date without timezone, and the db should save it as received. Mind opening an issue?

**Reply by hugopvl** [reply]
Something curious, time of the system where the directus runs:
![image](https://user-images.githubusercontent.com/81326296/125767309-d096bd59-cd29-4d65-8203-f9823dfc0af7.png)

time displayed on firefox console:
![image](https://user-images.githubusercontent.com/81326296/125767476-14a50789-7d44-45c5-8d16-52a0c50f3dfa.png)

where does directus get that hour?

**Reply by rijkvanzanten** [reply]
Good question! Directus itself doesn't set any timezones anywhere, so it could be a server problem 🤔
